account_chatter/views/account_move_views.xml

```diff
@@ -1,30 +1,23 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
 
-    <record id="view_move_form_show_chatter" model="ir.ui.view">
+    <record id="view_move_form" model="ir.ui.view">
         <field name="name">account.move.form.inherit.account.chatter.show</field>
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
-        <field name="groups_id" eval="[(4, ref('account_chatter.group_show_account_chatter_notifications'))]" />
```

Comment:
Are you sure this doesn't work anymore?

Is ee the field [still exists in v16](https://github.com/odoo/odoo/blob/79dc6384329b1e8bd2338160d62028ac0c1cdce4/odoo/addons/base/models/ir_ui_view.py#L263), so it should be possible